docs/standard-library/built-in-decorators.md

```diff
@@ -531,6 +531,23 @@ model Dog extends Pet  {kind: "dog", bark: boolean}
 ```
 
 
+### `@dateFormat` {#@dateFormat}
+
+
+```typespec
+dec dateFormat(target: zonedDateTime | ModelProperty, format: rfc1123 | rfc7231 | unixTimeStamp | string)
```

Comment:
The values should be rfc3339 and rfc7231.  rfc7231 is the replacement for rfc1123